idl/group0_state_machine.idl.hh

```diff
@@ -32,7 +32,7 @@ struct write_mutations {
 };
 
 struct group0_command {
-    std::variant<service::schema_change, service::broadcast_table_query, service::topology_change, service::write_mutations> change;
+    std::variant<service::schema_change, service::broadcast_table_query, service::topology_change, service::mixed_change, service::write_mutations> change;
```

Comment:
Adding variant options in the middle is backwards incompatible, you need to add at the end

Reply:
I followed your advice, but frankly - I don't get it, can you please elaborate what _backward compatibility_ do you have in mind?

Reply:
since we do ser::serialize  on a command and then potentially send it over the network perhaps it's about that, on mixed version cluster it could decode incorrectly

Reply:
> since we do ser::serialize on a command and then potentially send it over the network perhaps it's about that, on mixed version cluster it could decode incorrectly

true.

@ptrsmrn please fix

Reply:
OK, thanks, fixed.